mtp_noms_ops/apps/security/views/check.py

```diff
@@ -17,7 +17,7 @@ class CheckListView(SecurityView):
     form_class = CheckListForm
 
 
-class ActionCheckView(FormView):
+class BaseAcceptOrRejectCheckView(FormView):
```

Comment:
is there only one subclass now? could merge into one if so?

Reply:
Yep, sorted. I misread a title reassignment, which was actually just for the breadcrumbs